src/main/scala/viper/gobra/translator/encodings/slices/SliceEncoding.scala

```diff
@@ -215,31 +206,42 @@ class SliceEncoding(arrayEmb : SharedArrayEmbedding) extends LeafTypeEncoding {
     }
   }
 
-
   /**
     * Obtains permission to all cells of a slice
-    * getCellPerms[loc: []T] -> forall idx :: {loc(a, idx) 0 <= idx < cap(l) ==> Footprint[ loc[idx] ]
+    *     getCellPerms[loc: []T] ->
+    *         forall idx :: { loc(a, idx) } 0 <= idx < cap(l) ==> Footprint[ loc[idx] ]
```

Comment:
```suggestion
    *         forall idx :: { loc(a, idx) } 0 <= idx < bound ==> Footprint[ loc[idx] ]
    *            where bound is len(l) if sliceBound = SliceBound.Len, and cap(l) otherwise.
```